Source/Field.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace GameOfLife.Source
+{
+    /// <summary>
+    /// Class for new fields to be created.
+    /// </summary>
+    public class Field
+    {
+        /// <summary>
+        /// Property used to store current field width.
+        /// </summary>
+        public ushort FieldWidth { get; protected set; }
+        /// <summary>
+        /// Property used to store current field height.
+        /// </summary>
+        public ushort FieldHeight { get; protected set; }
```

Comment:
> There is no need for FieldWidth and FieldHeight as the only place where it is used is to set up console size, so you can just pass these two value to that method specifically instead of creating public properties, except if it is used outside of this class, but then again need to thing if that is necessary.

fixed